### DISCUSSION TITLE
Performance Testing - Importance of Unique Users vs. Unique Sessions

### DISCUSSION THREAD

**Discussion by trussrod75** [OpenIDC/mod_auth_openidc#673] [Q&A] [answered]
We're planning our performance testing of our Apache instances with mod_auth_openidc.  Is the number of unique users a factor where mod_auth_openidc is concerned?  Is there a difference in overhead, etc. of the module if we have 1 user w/ 100 concurrent, unique sessions or 100 unique users with 100 concurrent sessions?

OIDCSessionType is default (server-cache)
Cache settings are all default.

Persistence is enabled on lb so that a user is always directed to the same web server node for the duration of thier session (based on a lb set cookie).

Any feedback you can provide would be greatly appreciated!

**Comment by zandbelt** [comment, marked as answer]
there's no difference in overhead, the concept of a user is not really visible to mod_auth_openidc, the primary concept is a session

**Reply by trussrod75** [reply]
Thanks!
